BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -2432,15 +2443,19 @@ IEnumerator CruiseRoutine()
                     while (emitter.MoveNext())
                     {
                         if (emitter.Current == null) continue;
+                        /*
                         if (!hasRCS)
                         {
-                            emitter.Current.sizeGrow = Mathf.Lerp(emitter.Current.sizeGrow, 0, 20 * Time.deltaTime);
+                            emitter.Current.sizeGrow = Mathf.Lerp(emitter.Current.sizeGrow, 0, 20 * Time.deltaTime); //uh, why? this turns reasonable missileFX into giant doom plumes
```

Comment:
That would be lerping `sizeGrow` to 0 from the initial value of 99999 (i.e., the initial booster grow effect?), which would prevent exponentially growing plumes, wouldn't it? Though I could see it breaking plumes that aren't designed to grow like that. I think as long as removing it doesn't break any ancient missiles, doing so should be fine.

In any case, I managed to trace it back to 2014 where it was originally added in commit f968787e44 as
```
f968787e44 (Paolo Encarnacion 2014-10-08 21:34:26 -0400 311)          pe.sizeGrow = Mathf.Lerp(pe.sizeGrow, 1, 0.3f);
```